### PR TITLE
Add support for OTEL_SERVICE_NAME

### PR DESCRIPTION
**Pull request by ocelotl**
Merge after #108

### CODE REVIEW

**Review by codeboten — COMMENTED** [review]
Please rebase

**Comment by ocelotl** [comment]
> Please rebase

done

File: tests/test_configuration.py [hunk 1]
```diff
@@ -123,7 +123,7 @@ def test_only_service_name_and_token(self, mock_otlp_span_exporter):
         # The worker thread in MockBatchSpanProcessor is configured to
         # wait 1ms before exporting. Sleeping here to give enough time to the
```
Comment (codeboten) [inline tests/test_configuration.py:124]: the comment should be updated to reflect the change

**Review by codeboten — APPROVED** [review]
Just a minor comment about a comment :) Otherwise looks good

File: tests/test_configuration.py (hunk 1 above)
Comment (ocelotl) [reply tests/test_configuration.py]: Fixed

**Pull request merged**